CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contribution Guidelines
+
+## Issues and Feature Requests
+
+If you got a question, bug report, or feature request, submit an issue to [our repository](https://github.com/DevExpress/devextreme-reactive) but please check the [list of active issues](https://github.com/DevExpress/devextreme-reactive/issues) first to avoid duplicates. 
```

Comment:
"submit an issue" seems like you are submitting a problem, something negative. Perhaps 'submit a ticket'?

Reply:
There is no such thing as a ticket on GitHub, only issues and pull requests

Reply:
```suggestion
If you got a question, bug report, or feature request, submit an issue to [our repository](https://github.com/DevExpress/devextreme-reactive). You should check the [list of active issues](https://github.com/DevExpress/devextreme-reactive/issues) first to avoid duplicates. 
```

Reply:
Ok. If you got a question, bug report, or feature request, submit an issue to [our repository](https://github.com/DevExpress/devextreme-reactive). You should check the [list of active issues](https://github.com/DevExpress/devextreme-reactive/issues) first to avoid duplicates.